src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml

```diff
@@ -57,6 +57,38 @@
                 </Style.Triggers>
             </Style>
 
+            <Style x:Key="ValidityIndicatorStyle" TargetType="{x:Type imaging:CrispImage}">
```

Comment:
This can be simplified.
First, you need data trigger only for one of the values. The other one can be default.
Second, you can move data triggers directly into the `EditConnectionDataTemplate`:
```
<Style x:Key="ValidityIndicatorStyle" TargetType="{x:Type imaging:CrispImage}">
  <Style.Setters>
    <Setter Property="Width" Value="16" />
    <Setter Property="Height" Value="16" />
    <Setter Property="HorizontalAlignment" Value="Left" />
    <Setter Property="VerticalAlignment" Value="Top" />
    <Setter Property="Margin" Value="0,0,4,0" />
    <Setter Property="Moniker" Value="{x:Static imagecatalog:KnownMonikers.StatusOKOutline}"/>
  </Style.Setters>
</Style>
```
```
<imaging:CrispImage x:Name="NameValidityIndicator" Grid.Row="0" Style="{StaticResource ValidityIndicatorStyle}" />
<imaging:CrispImage x:Name="PathValidityIndicator" Grid.Row="1" Style="{StaticResource ValidityIndicatorStyle}" />
```
```
                <DataTemplate.Triggers>
                    <DataTrigger Binding="{Binding IsNameValid}" Value="False">
                        <Setter TargetName="NameValidityIndicator" Property="Moniker" Value="{x:Static imagecatalog:KnownMonikers.StatusRequiredOutline}"/>
                    </DataTrigger>
                    <DataTrigger Binding="{Binding IsPathValid}" Value="False">
                        <Setter TargetName="PathValidityIndicator" Property="Moniker" Value="{x:Static imagecatalog:KnownMonikers.StatusRequiredOutline}"/>
                    </DataTrigger>
                </DataTemplate.Triggers>
```